php-checks/src/main/java/org/sonar/php/checks/formatting/ControlStructureSpacingCheck.java

```diff
@@ -49,20 +48,20 @@ private void checkForeachStatement(FormattingStandardCheck formattingCheck, AstN
     AstNode asKeyword = foreachExpr.getFirstChild(PHPKeyword.AS);
     AstNode doubleArrow = foreachExpr.getFirstChild(PHPPunctuator.DOUBLEARROW);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "doubleArrow". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1854)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "doubleArrow" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)